### ISSUE TITLE
Python 3.12 blocked by failing config checks

### ISSUE THREAD

**Issue by Scienfitz** [emdgroup/baybe#156]
The tests that ensure an invalid config is checked failes in python 3.12 (not in others)

seems related to `cattrs`

also, it is possible to write a config with `recomEnDer` (spelling mistake) and it will take a default instead of throwing an error

PR for Python 3.12 upgrade: #153

**Comment by AdrianSosic** [comment]
Hi @Scienfitz, the 3.12 issue is now solved (according to the issue description), but in the text, you mention a separate unrelated problem. Can you create a separate issue for it, ideally with a minimal example, and then resolve this one here?

**Comment by Scienfitz** [comment]
> Hi @Scienfitz, the 3.12 issue is now solved (according to the issue description), but in the text, you mention a separate unrelated problem. Can you create a separate issue for it, ideally with a minimal example, and then resolve this one here?

I'll close this Issue and put the other topic on dev agenda

**Comment by Scienfitz** [comment]
Closing as this Issue was fixed in #153
